### PR TITLE
Update Intercom-cohort.md

### PR DESCRIPTION
**Pull request by sina-jamshidi**
Updating cohort export docs with important information for customers

# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/destinations/intercom-cohort.md [hunk 1]
```diff
@@ -28,4 +28,8 @@ Send behavioral cohorts from Amplitude to Intercom so that you can better engage
 
 !!!note
     
-    For scheduled cohort syncs, the initial sync includes the full cohort. All subsequent syncs update the original cohort via additions and removals.
\ No newline at end of file
+    Users are exported based on their **User Id**. Users must already exist on Intercom with corresponding User Ids in order to be properly synced
```
Comment (caseyamp) [inline docs/data/destinations/intercom-cohort.md:31]:
```suggestion
    Users are exported based on their **user ID**. Users must already exist in Intercom with corresponding user IDs in order to be properly synced
```

**Review by caseyamp — CHANGES_REQUESTED** [review]
Just a little nit for style guide, thanks @sina-jamshidi

File: docs/data/destinations/intercom-cohort.md (hunk 1 above)
Comment (sina-jamshidi) [reply docs/data/destinations/intercom-cohort.md:31]: Great thanks Casey!

**Review by caseyamp — APPROVED** [review]

**Comment by caseyamp** [comment]
@sina-jamshidi merge when ready!

**Pull request merged**